rust/agents/relayer/src/relayer.rs

```diff
@@ -188,6 +188,8 @@ impl BaseAgent for Relayer {
                 self.validator_announce.clone(),
                 self.allow_local_checkpoint_syncers,
                 self.core.metrics.clone(),
+                0,
+                5,
```

Comment:
should this be configurable?

Reply:
I don't feel strongly. Would suggest the quick and dirty way for now and we can adjust later if necessary. The less config surface area the better.